pip/main.ts

```diff
@@ -137,10 +162,15 @@ function parseDryRun(output: string): [object] {
   // Parse dependency names and versions.
   const deps = new_deps.split(' ');
   for (var i = 0; i < deps.length; i++) {
+    const pkg = deps[i];
```

Comment:
`charset-normalizer-<ver>` caused issues with the previous logic as it split on the _first_ `-`. We need to split on the _last_ dash.